examples/docs/src/Examples/Docs/Aggregations/Bucket/AdjacencyMatrixAggregation.php

```diff
@@ -12,7 +12,7 @@
  *
  * Class: AdjacencyMatrixAggregation
  *
- * Date: 2019-08-05 08:49:19
+ * Date: 2019-08-06 06:56:02
```

Comment:
Is the date important here? If we run the generator and the code is the same we'll have different files with just a date diff.  This is something expected or can we remove it to avoid superfluous commits?

Reply:
I do like the idea of keeping the diff small!

Reply:
no it's not, you are 100% correct on the diff, that bit me already a couple of times. will remove it